### PR TITLE
Translate few HTML resources to Mandarin

### PR DESCRIPTION
**Pull request by lilyJi**

### CODE REVIEW

**Review by daniel-beck — APPROVED** [review]
Google Translate looks reasonable, and I don't think we have people who speak Mandarin on the team.

**Comment by daniel-beck** [comment]
@fengxx WDYT?

File: war/src/main/webapp/help/project-config/block-downstream-building_zh_CN.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div>
+  启用选项后，Jenkins会阻止该项目的建立如果该项目的子项目在队列中或正在构建。
```
Comment (fengxx) [inline war/src/main/webapp/help/project-config/block-downstream-building_zh_CN.html]: 选中此选项后，当该项目的子项目在队列或构建中时，Jenkins将阻止该项目构建

File: war/src/main/webapp/help/system-config/master-slave/demand/idleDelay_zh_TW.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <div>
     Slave 真正離線前停留在閒置狀態的分鐘數。
-</div>
```
Comment (fengxx) [inline war/src/main/webapp/help/system-config/master-slave/demand/idleDelay_zh_TW.html]: by accident?
change EOF for unrelated files may cause merge issues.

Comment (fengxx) [inline war/src/main/webapp/help/system-config/master-slave/demand/idleDelay_zh_TW.html]: it is traditional chinese, not related to this PR

**Comment by oleg-nenashev** [comment]
Needs fix according to the feedback from @fengxx

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
@lilyJi any plans to address comments from @fengxx ?

**Comment by oleg-nenashev** [comment]
@lilyJi @fengxx @daniel-beck there is no progress since the last feedback + there are merge conflicts due to the changes made by @linuxSuRen. I propose to close this PR

**Comment by LinuxSuRen** [comment]
It does been long time ago.

**Comment by oleg-nenashev** [comment]
@LinuxSuRen are you fine with closing?

**Comment by LinuxSuRen** [comment]
@oleg-nenashev I can help put these translation into repo, if the orignal author don't focus it.

**Pull request closed without merging**